packages/app-store/routing-forms/lib/evaluateRaqbLogic.test.ts

```diff
@@ -6,7 +6,7 @@ import { evaluateRaqbLogic, RaqbLogicResult } from "./evaluateRaqbLogic";
 vi.mock("../components/react-awesome-query-builder/widgets", () => ({
   default: {},
 }));
-vi.mock("@calcom/ui");
+vi.mock("@calcom/ui", () => ({}));
```

Comment:
Because @calcom/ui is being imported in config.tsx now we need to mock it to ensure no UI component is included as that somehow causes formbricks to execute which crashes the tests.